git-worktree/tests/copy_index/mod.rs

```diff
@@ -0,0 +1,91 @@
+use crate::{dir_structure, fixture_path};
+use git_object::bstr::ByteSlice;
+use git_odb::FindExt;
+use git_worktree::{copy_index, Options};
+use std::fs;
+
+#[cfg(unix)]
+use std::os::unix::prelude::MetadataExt;
+
+type Result<T = ()> = std::result::Result<T, Box<dyn std::error::Error>>;
```

Comment:
This should be on `crate` level, this is a module. Usages should be `crate::Result`;